jui-grid/index.d.ts

```diff
@@ -0,0 +1,996 @@
+// Type definitions for jui-grid 2.0
+// Project: https://github.com/juijs/jui-grid
+// Definitions by: Jin-Ho Park <https://github.com/easylogic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import {UIEvent} from 'jui-core';
+
+export interface GridColumn {
+    (index: number|string): this;
```

Comment:
If `GridColumn` is a value s.t. you can call `new GridColumn(0)`, you should use:
```ts
export class GridColumn {
    constructor(index: number | string);
}
```
(Be sure to add a test for it!)